docs/agile-workflow/README.md

```diff
@@ -25,3 +26,31 @@ The Agile Workflow for Cloud Service Certification falls into two main work stre
 
 - Pull Requests and Issues raised by the FINOS Community are automatically assigned to CSC Project Maintainers and allocated to the CSC project backlog for sprint prioritisation at the next scheduled sprint planning session.
 - Critical Pull Requests and Issues are allocated to the CSC project team for immediate fulfilment at the discretion of the CSC Project Maintainers.
+
+## <a name="reviews">CSC Asynchronous Pull Request and Code Reviews</a>
+
+Cloud Service Certification openly encourages and invites asynchronous pull request and code reviews as part of its agile delivery workflow from the CSC and wider FINOS Community without direct invitation from CSC maintainers or contributors in order to increase pace and efficiency within the project.
+
+### Reviewing CSC Pull Requests
+
+Asynchronous reviews can take place in any Cloud Service Certification pull request that has a `ready for review` label assigned as illustrated below.
+
+<img alt="Open Pull Request to Review" src="../_images/review-1.png" width="60%">
+
+### Inline Code Reviews and Comments
+
+Inline code reviews and comments are very much encouraged, invited and welcome in order to give context and instructions to the pull request contributor as demonstrated in the screenshot below.
+
+<img alt="Add Files Changed Inline Comments" src="../_images/review-2.png" width="60%">
+
+### Reviewing Changes and Giving Feedback
+
+Reviews and feedback from the CSC and wider FINOS Community are also welcome by submitting your review using the `Review Changes` feature as illustrated below.
+
+<img alt="Submit Pull Request Review" src="../_images/review-3.png" width="60%">
+
+By submitting asynchronous reviews you will be recognised as a Cloud Service Certification open source contributor and will be increasing the quality and pace of work being contributed into the FINOS CSC project and wider financial services industry.
+
+### Receiving Community Feedback
```

Comment:
@TLATER - I have added the following lines to resolve your feedback here ...

https://github.com/finos/cloud-service-certification/pull/137#pullrequestreview-726365294
> This LGTM :)
> 
> _Perhaps_ we could explicitly add a section explaining that authors are expected to take into account feedback from external contributors as well, just to make it clear from both sides of the coin, but I don't actually think we need that much formalism.